.github/workflows/docker-push-to-registries.yml

```diff
@@ -140,8 +140,9 @@ jobs:
 
   scan-image:
     needs: build
-    uses: Telicent-oss/shared-workflows/.github/workflows/trivy-image-scan.yml@main
+    uses: Telicent-oss/shared-workflows/.github/workflows/trivy-image-scan.yml@TELFE-186/vulv-scan-report-on-release
```

Comment:
Needs to pin back to `main` before merging

Reply:
🦅 👀